src/interface/storage.thrift

```diff
@@ -513,6 +509,6 @@ service StorageService {
     GetUUIDResp getUUID(1: GetUUIDReq req);
 
     // Interfaces for edge and vertex index scan
-    LookUpVertexIndexResp lookUpVertexIndex(1: LookUpIndexRequest req);
-    LookUpEdgeIndexResp   lookUpEdgeIndex(1: LookUpIndexRequest req);
+    LookUpIndexResp   lookUpVertexIndex(1: LookUpIndexRequest req);
+    LookUpIndexResp   lookUpEdgeIndex(1: LookUpIndexRequest req);
```

Comment:
Overall looks good to me. 
Would be more perfect if `lookUpVertexIndex` and `lookUpVertexIndex` unify into `lookupIndex`